Week2/homework/maartjes-work.js

```diff
@@ -43,19 +43,28 @@ const tuesday = [
 ];
 
 const maartjesTasks = monday.concat(tuesday);
+
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const mappedTasks = tasks.map(task => task.duration / 60);
+  const filteredTasks = mappedTasks.filter(task => task >= 2);
+  const multiplyTasks = filteredTasks.map(task => task * hourlyRate);
+  const roundedTasks = multiplyTasks.map(task => task);
```

Comment:
This line is complete useless. The `roundedTasks` array has the same values as the `multiplyTasks` array, because you are mapping each element onto itself. Furthermore, these names are incorrect because what you call `multiplyTasks` are actually `hours`.